$OpenBSD: patch-lenses_simplelines_aug,v 1.4 2015/06/06 10:59:27 jasper Exp $

- From 8251eb3e182fb5c196fdeac896e7a06b9bb7a819 Mon Sep 17 00:00:00 2001
  From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
  Date: Tue, 2 Jun 2015 10:13:21 +0200
  Subject: [PATCH] Simplelines: parse OpenBSD's hostname.if(5) files
- rc.conf* are not shell scripts anymore.

--- lenses/simplelines.aug.orig	Fri Nov  2 16:20:11 2012
+++ lenses/simplelines.aug	Tue Jul 22 02:05:54 2014
@@ -43,5 +43,8 @@ let filter = incl "/etc/at.allow"
            . incl "/etc/cron.d/cron.deny"
            . incl "/etc/default/grub_installdevice"
            . incl "/etc/pam.d/allow.pamlist"
+           . incl "/etc/hostname.*"
+           . incl "/etc/rc.conf"
+           . incl "/etc/rc.conf.local"
 
 let xfm = transform lns filter
